test/uk/gov/hmrc/cardpaymentfrontend/controllers/PaymentFailedControllerSpec.scala

```diff
@@ -33,127 +36,93 @@ class PaymentFailedControllerSpec extends ItSpec {
       }
 
-      "render the page without content line 4 for Pngr Tax Types in English" in {
-        val result = systemUnderTest.renderPage2()(fakeGetRequest)
+      "render the correct content in English for origins with no OpenBanking" in {
+        // TODO: May need changing if/when ItSa payment methods are changed
+        PayApiStub.stubForFindBySessionId2xx(TestJourneys.ItSa.journeyAfterFailWebPayment)
+        val result = systemUnderTest.renderPage(fakeGetRequest)
         val document = Jsoup.parse(contentAsString(result))
-        document.selectXpath("//*[@id=\"main-content\"]/div/div/ul/li[3]").text() shouldNot contain("the address you gave does not match the one your card issuer has")
+        val title = document.body().select(".govuk-heading-xl")
+        title.select("h1").text() shouldBe "Payment failed"
+        println(document)
+        document.getElementById("sub-heading").text() shouldBe "No payment has been taken from your card."
+        document.getElementById("line1").text() shouldBe "The payment may have failed if:"
+        document.getElementById("line2").text() shouldBe "there are not enough funds in your account"
+        document.getElementById("line3").text() shouldBe "you entered invalid or expired card details"
+        document.getElementById("line4").text() shouldBe "the address you gave does not match the one your card issuer has"
       }
 
-      "render the page without content line 4 for Pngr Tax Types in Welsh" in {
-        val result = systemUnderTest.renderPage2()(fakeGetRequestInWelsh)
+      "render the correct content in Welsh for origins with no OpenBanking" in {
+        // TODO: May need changing if/when ItSa payment methods are changed
+        PayApiStub.stubForFindBySessionId2xx(TestJourneys.ItSa.journeyAfterFailWebPayment)
+        val result = systemUnderTest.renderPage(fakeGetRequestInWelsh)
         val document = Jsoup.parse(contentAsString(result))
-        document.selectXpath("//*[@id=\"main-content\"]/div/div/ul/li[3]").text() shouldNot contain("nid yw’r cyfeiriad a roesoch i ni’n cyd-fynd â’r un sydd gan ddosbarthwr eich cerdyn")
+        val title = document.body().select(".govuk-heading-xl")
+        title.select("h1").text() shouldBe "Taliad wedi methu"
+        println(document)
+        document.getElementById("sub-heading").text() shouldBe "Nid oes taliad wedi’i dynnu o’ch cerdyn."
+        document.getElementById("line1").text() shouldBe "Mae’n bosibl bod y taliad wedi methi oherwydd:"
+        document.getElementById("line2").text() shouldBe "nid oes yna ddigon o arian yn eich cyfrif"
+        document.getElementById("line3").text() shouldBe "rydych wedi nodi manylion cerdyn sy’n annilys neu sydd wedi dod i ben"
+        document.getElementById("line4").text() shouldBe "nid yw’r cyfeiriad a roesoch i ni’n cyd-fynd â’r un sydd gan ddosbarthwr eich cerdyn"
       }
 
     }
```

Comment:
There are no tests for the submission of the form on this page. Please add them.